src/Mobs/Squid.cpp

```diff
@@ -9,7 +9,7 @@
 
 
 cSquid::cSquid(void) :
-	super("Squid", mtSquid, "entity.squid.hurt", "entity.squid.death", 0.95, 0.95)
+	super("Squid", mtSquid, "entity.squid.hurt", "entity.squid.death", "", 0.95, 0.95)
```

Comment:
Do squids not have an ambient sound?

Reply:
Seems like it was introduced in 1.13. I'll add the sound name.